### ISSUE TITLE
On version v1.3.5 android cannot run. Execution failed for task ':app:mergeExtDexDebug'. (jetified-android-gif-drawable-1.2.29-runtime.jar.)

### ISSUE THREAD

**Issue by heyheythere** [liodali/osm_flutter#573]
The package functions correctly on iOS, but I encountered issues running it on Android.

Flutter doctor:
[✓] Flutter (Channel stable, 3.24.0, on macOS 15.1 24B83 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 16.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.2)
[✓] VS Code (version 1.95.3)
[✓] Connected device (4 available)
[✓] Network resources

Kotlin version: 1.9.23

distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-all.zip

Error Summary
When attempting to build the Flutter application, the build fails with the following error related to android-gif-drawable:

```
ERROR:/Users/heythere/.gradle/caches/transforms-3/306f68d744e81358add969a0c803bb4b/transformed/jetified-android-gif-drawable-1.2.29-runtime.jar: D8: java.lang.NullPointerException

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform android-gif-drawable-1.2.29.aar (pl.droidsonroids.gif:android-gif-drawable:1.2.29) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=21, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: /Users/heythere/.gradle/caches/transforms-3/306f68d744e81358add969a0c803bb4b/transformed/jetified-android-gif-drawable-1.2.29-runtime.jar.
         > Error while dexing.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 16s
Error: Gradle task assembleDebug failed with exit code 1

```

 I would greatly appreciate your help. Thank you!

**Comment by liodali** [comment]
i will make hotfix for it tonight or tomorrow

**Comment by liodali** [comment]
hotfix has been published `1.3.5+1`

**Comment by heyheythere** [comment]
Thank you! The build was successful.
